facebook-news-bot.service

```
@@ -0,0 +1,17 @@
+[Unit]
+Description=facebook-news-bot
+
+[Service]
+User=content-api
+Group=content-api
+Restart=on-failure
```

This should be `Restart=no`. If for some reason the service can't start we want it to fail fast and default to the auto scaling group to take the instance out of service and replace with a new one.